src/averageAges.js

```diff
@@ -55,7 +68,16 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const sort = people.filter(person => onlyWithSon
```

Comment:
```suggestion
  const mothers = people.filter(person => onlyWithSon
```
Why sort? Let's make it clear